src/js/rx/containers/Active.jsx

```diff
@@ -1,17 +1,73 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import { browserHistory } from 'react-router';
 
 import PrescriptionList from '../components/PrescriptionList';
+import PrintList from '../components/PrintList';
+import SortMenu from '../components/SortMenu';
+
 import { loadData } from '../actions/prescriptions.js';
 
 class Active extends React.Component {
   componentWillMount() {
     this.props.dispatch(loadData());
+    this.handleSortOnChange = this.handleSortOnChange.bind(this);
+    this.handleSortOnClick = this.handleSortOnClick.bind(this);
+    this.dispatchSortAction = this.dispatchSortAction.bind(this);
+  }
+
+  dispatchSortAction(action) {
+    this.props.dispatch({ type: action });
   }
+
+  handleSortOnChange(domEvent) {
+    if (domEvent.type === 'change') {
+      browserHistory.push({
+        pathname: '/rx',
+        query: {
+          sort: domEvent.target.value
+        }
+      });
+    }
+    this.dispatchSortAction(domEvent.target.value);
+  }
+
+  handleSortOnClick(domEvent) {
+    const fullURL = domEvent.target.href;
+
+    // Find the sort parameter, split the query string on the = and retrieve value
+    const sortParam = fullURL.match(/sort=[_A-Z]{1,}/)[0].split('=')[1];
```

Comment:
Looks like it's a jQuery plugin I was thinking of, not jQuery core. 

There is a WHATWG proposal to actually make parsing a URL part of the JS standard (why it was not from the start, who knows), but apparently, it's not supported in Safari or IE: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams We can probably find a polyfill for it though.